corpus/literals.txt

```diff
@@ -197,57 +198,68 @@ Lists
 =====================================
 
 [a, b, [c, d]]
+[*()]
+[*[]]
+[*a]
+[*a.b]
+[*a[b].c]
```

Comment:
@maxbrunsfeld there are some cases I've added for splats that were showing up as parse errors, and makes our handling of lists, dictionaries, and sets more robust.